docs/workflow_solutions/using_anaconda.md

```diff
@@ -282,6 +282,94 @@ Now we can be sure that the correct versions of the software will be installed o
     The example above is provided only for illustration purposes. The error has since been fixed, but the example above really happened and is helpful to explain version pinning.
 <!-- markdownlint-enable MD046 -->
 
+#### Good Practice for Finding Software Packages on Anaconda
+
+Finding Anaconda software packages involves searching through the available “Channels” and repositories to locate the specific packages that contain functions that you need for your environment. Channels are locations that store packages, these locations are where Anaconda looks for packages when an installation is to be done. The following are Anaconda Channels that are readily used to house majority of the packages used in scientific research. i. Anaconda ii. Conda-Forge iii. BioConda. There are other Channels also. If you want more information on Anaconda Channels please see their [docs](https://docs.anaconda.com/).
+
+##### Step-by-Step Guide to Finding Anaconda Software Packages
+
+If we find the package at one of the Channel sources mentioned above, we can check the Platform version to ensure it is either "noarch" (if available) or linux. After noting the version, we can click the "source" or "repo" link (if available) or "homepage". Then we try to find the latest version. For a package found on GitHub, click "Releases" on the right-hand side. Verify that the latest Release is the same as, or very close to, the version on Anaconda or Pypi. If so, the package is being maintained on Anaconda/Pypi and suitable for use. Note the exact software name, version, and channel (if not on Pypi).
```

Comment:
This is great and appropriately terse.

Write it as "PyPI". That is how it is written on their site: https://pypi.org/. Stands for "(Py)thon (P)ackage (I)ndex".